sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -400,4 +400,21 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with Logging {
         -> "1.000000000000000000"
     )
   }
+
+  test("SPARK-27667 test cli prompt using spark.cli.print.current.db") {
+    runCliWithin(1.minute, Seq("--conf", "spark.sql.cli.show.current.db=true"))(
+      "show tables;" -> "spark-sql (default)",
+      "CREATE DATABASE hive_test_db;"
+        -> "",
+      "USE hive_test_db;"
+        -> "spark-sql (hive_test_db)"
```

Comment:
Since these are short, one line will be enough.
```scala
-      "CREATE DATABASE hive_test_db;"
-        -> "",
-      "USE hive_test_db;"
-        -> "spark-sql (hive_test_db)"
+      "CREATE DATABASE hive_test_db;" -> "",
+      "USE hive_test_db;" -> "spark-sql (hive_test_db)"
```